packages/liveblocks-react-comments/src/components/InboxNotification.tsx

```diff
@@ -374,63 +372,62 @@ const InboxNotificationThread = forwardRef<
     //       it's not a big deal, the only scenario where it would be superfluous would be if the user provides their own
     //       `href` AND disables room names in the title via `showRoomName={false}`.
     const { info } = useRoomInfo(inboxNotification.roomId);
-    const { unread, date, aside, title, content, threadId, commentId } =
```

Comment:
Nothing has been changed in this `useMemo` code snippet. I only removed `threadId` from the return value because we no longer require it.